lib/ansible/modules/cloud/amazon/aws_organization_units.py

```diff
@@ -202,6 +201,10 @@ def _create_ou(self, name, parent_id):
         else:
             return ou['OrganizationalUnit']
 
+    # Name can be full ou path in form "root/parent/child"
+    # leading, and ending "/" will be stripped to handle
+    # "/root/child" or "root/parent/" inputs
+    # Parent ous will be recursively dereferenced
     def create_ou(self, name):
```

Comment:
Rather than adding this as a comment here, it would be really good if you could add it to the description up on line 59.

It's obvious to you, but if I used the ARN, it wouldn't work...

Reply:
Would be nice if you accepted ARNs too ;)

Reply:
Let me see what I can do

Reply:
I've refactored the code and added support of ARN